### ISSUE TITLE
feature request: use vscode config for formatting inline template

### ISSUE THREAD

**Issue by sandangel** [onixie/angular-component-extension#4]
I had add vscode config html "html.format.wrapAttributes": "force-aligned", but the formatter ignored it and every attribute just go on 1 line make template hard to read.

Can you add support use vscode html config to format inline template?

**Comment by onixie** [comment]
It's a bit tricky, cuz I use vscode-xml internally to format inline template. And as you can see it is for xml, and html settings are totally ignored. I'll consider how to make it, but maybe a little bit time needed.

**Comment by sandangel** [comment]
don't worry, this is an open source project. In the meantime can you tell me why we don't use vscode html formatter? Is it feasible to replace vscode-xml with something like vscode-html? And it would be great if we can have formatOnSave feature.

**Comment by onixie** [comment]
yes, I admit html can be more appropriate than xml on this subject, though templates are often not valid html.
Actually the reason to choose vscode-xml is quite accidental. The html formatter was more troublesome  when dealing with bindings in template. Since bindings are js-like expressions, people can write code like `*ngIf="1 < 2"` which make html formatter paralysis. BTW, the xml formatter cannot help the case either, but work a bit better.

P.S., there are some other html formatter extensions in marketplace, but even worse than the default one .

formatOnSave can be another feature, I do think it is useful.
